### PR TITLE
Issue 34 35

### PR DESCRIPTION
**Pull request by robsongajunior**
- [*] update - 0.0.14 -> 0.0.15
- [-] delete  - partials/components [death files]
- [-] delete  - font-face files type css/demo/etc [death files]
- resolve - issue #34 #35

### CODE REVIEW

**Comment by robsongajunior** [comment]
fixed issues: #34 #35

File: partials/footer.html [hunk 1]
```diff
@@ -142,7 +143,7 @@
                     </div>
                 </aside>
                 <div class="js-footer">
-                    <a href="#"><img src="/images/icon-logo-js.png" alt="icon JS" title="Javascript"></a>
+                    <a href="https://www.javascript.com/" target="_blan"> <img src="/images/icon-logo-js.png" alt="JavaScript" title="javascript.com"> </a>
```
Comment (UltCombo) [inline partials/footer.html:146]: @robsongajunior @jaydson is this javascript.com link supposed to be here? Feels a bit out of place IMHO.

Comment (jaydson) [inline partials/footer.html:146]: Ahaaha No. WTF @robsongajunior

**Comment by robsongajunior** [comment]
hmmm ok, my bad, I had a wrong idea.
I will fix this.

cc/ @UltCombo  @jaydson

**Comment by robsongajunior** [comment]
fixed

**Pull request merged**